src/main/java/frc/robot/subsystems/SimulationSubsystem.java

```diff
@@ -19,6 +23,11 @@ public class SimulationSubsystem extends SubsystemBase {
   private final Field2d m_field;
 
   private final SwerveSubsystem swerveSubsystem;
+
+  private Translation2d swerveWheelOffset = new Translation2d(5,5);
```

Comment:
This puts them in sort of the middle of the field. Not sure if there's a better placement. (If only we could make another Field2d :/)